source/audio/utils.py

```diff
@@ -121,15 +122,33 @@ class AudioSessionCallback(DummyAudioSessionCallback):
 	_audioSessionEventListeners: set[_AudioSessionEventsListener] = field(default_factory=set)
 
 	def onSessionUpdate(self, session: AudioSession) -> None:
+		"""
+		Callback function to be implemented by the customer.
+		Will be called for all existing sessions and newly created sessions.
+		"""
 		pass
 
 	def onSessionTerminated(self, session: AudioSession) -> None:
+		"""
+		Callback function to be implemented by the customer.
+		Will be called when each session is terminated or when unregister() method is called.
+		"""
 		pass
 
 	def register(self, applyToFuture: bool = True):
+		"""
+			Registers this callback.
+			This will internally call onSessionUpdate() for all current audio sessions.
+			Additionally if applyToFuture is True, it will also call onSessionUpdate() for all newly created sessions
+			until this callback is unregistered.

```

Comment:
please de-indent these docstrings.

Reply:
I am indenting these docstrings because it's more convenient for blind folks to skip over these docstrings using IndentNav when they are indented like this. It takes just a single keystroke to skip over an indented block like this, whereas without indentation blind people would have to arrow through the entire docstring every time they need to read this function. With more and more blind folks using IndentNav for coding - including many NVDA contributors, can we please allow this style for convenience?

Reply:
We plan to lint the entire codebase with Ruff within 2024.3. Can you confirm that this style won't automatically be stripped when we do this? Give we want to keep styling consistent, could you propose a change to `codingStandards.md` if we can support this with Ruff?

Reply:
Ok, I de-indented these docStrings.